src/bindings/c/ov/src/CMakeLists.txt

```diff
@@ -0,0 +1,50 @@
+# Copyright (C) 2018-2022 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+#
+
+set(TARGET_NAME openvino_ov_c)
+
+file(GLOB SOURCES ${CMAKE_CURRENT_SOURCE_DIR}/*.cpp)
+file(GLOB HEADERS ${InferenceEngine_C_API_SOURCE_DIR}/include/*.h)
+
+# create library
+
+add_library(${TARGET_NAME} ${HEADERS} ${SOURCES})
+add_library(openvino::runtime::ov ALIAS ${TARGET_NAME})
+
+target_link_libraries(${TARGET_NAME} PRIVATE openvino)
+
+target_include_directories(${TARGET_NAME} PUBLIC
+    $<BUILD_INTERFACE:${InferenceEngine_C_API_SOURCE_DIR}/include>)
```

Comment:
Do you have dependencies to IE C API?

Reply:
There is no dependency, we will resolve it.